gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreWrapper.java

```diff
@@ -318,4 +365,40 @@ private KeyFactory initKeyFactory(String kind) {
         keyFactories.put(kind, keyFactory);
         return keyFactory;
     }
+
+    private List<Entity> readBigBulk(List<Key> keys) {
```

Comment:
It also requires JavaDocs, explaining the intention and some of GAE-specific details.